src/data/real/pi.lean

```diff
@@ -275,4 +275,42 @@ begin
                ... = 1 - (u k) + (u k)^(2*(k:ℝ)+1) : by { rw [← pow_succ' (U:ℝ) (2*k)], norm_cast },
 end
 
+open finset interval_integral
+
+/-- This theorem establishes the Wallis Product for `π`. Our proof is largely about analyzing
+  the behavior of the ratio of the integral of `sin x ^ n` as `n → ∞`.
+
+  The proof can be broken down into two pieces. First, we use integration by parts to obtain
+  a recursive formula for `∫ x in 0..π, sin x ^ (n + 2)` in terms of `∫ x in 0..π, sin x ^ n`.
+  From this we can obtain closed form products of `∫ x in 0..π, sin x ^ (2 * n)` and
+  `∫ x in 0..π, sin x ^ (2 * n + 1)` via induction. Next, we study the behavior of the ratio
+  `∫ (x : ℝ) in 0..π, sin x ^ (2 * k + 1)) / ∫ (x : ℝ) in 0..π, sin x ^ (2 * k)` and prove that
+  it converges to one using the squeeze theorem. The final product for `π` is obtained after some
+  algebraic manipulation.  -/
+theorem tendsto_prod_pi_div_two :
```

Comment:
I believe the docstring here is out of date, since much of the work described in it has now moved to `analysis.special_functions.integrals`.

Reply:
Although the content of the proof isn't here, I feel like there is still some merit to describing it. I've added a note that mentions that the work is mostly in `analysis.special_functions.integrals`.